operator/v1/types_ingress.go

```diff
@@ -125,6 +127,11 @@ type IngressControllerSpec struct {
 	//
 	// +optional
 	NodePlacement *NodePlacement `json:"nodePlacement,omitempty"`
+
+	// securitySpec specifies settings for securing an IngressController.
+	//
+	// +optional
+	SecuritySpec *IngressSecuritySpec `json:"securitySpec,omitempty"`
```

Comment:
I think @Miciah's suggestion was to skip the wrapper type:

```go
SecurityProfile *configv1.SecurityProfileSpec `json:"securityProfile"`
```

which I'd be fine with

Reply:
I have updated to:
```
SecurityProfile *configv1.SecurityProfileSpec `json:"securityProfile,omitempty"`
```